edx_arch_experiments/scripts/generate_code_owner_mappings.py

```diff
@@ -166,7 +166,11 @@ def _map_repo_apps(csv_type, repo_csv, app_to_repo_map, owner_map, owner_to_path
         csv_repo_to_owner_map[row.get('repo url')] = owner
 
     for app, repo_url in app_to_repo_map.items():
-        owner = csv_repo_to_owner_map.get(repo_url, None)
+        # special case: enterprise-quokkas owns integrated channels but not the rest of edx-enterprise
+        if app == "integrated_channels":
```

Comment:
If you are going to hard code, could we at least add `EDX_REPO_APPS_OVERRIDES` or something like that so all the configs are together and this isn't buried in the code? The data structure would make it simpler to add others, but that is not the reason. It is difficult to figure out where this mapping code happens, and burying in the code itself just makes it that much harder.